src/FTPC.jl

```diff
@@ -210,32 +210,32 @@ end
 # Library initializations
 ##############################
 
-@doc """
-Global libcurl initialisation
-""" ->
+"""
+    ftp_init()
+
+Initialise global libcurl
+"""
 ftp_init() = curl_global_init(CURL_GLOBAL_ALL)
 
-@doc """
-Global libcurl cleanup
-""" ->
+"""
+    ftp_cleanup()
+
+Cleanup global libcurl.
+"""
 ftp_cleanup() = curl_global_cleanup()
 
 
 ##############################
 # GET
 ##############################
 
-@doc """
-Download file with non-persistent connection.
+"""
+    ftp_get(options::RequestOptions, file_name::AbstractString, save_path::AbstractString=""; mode::FTP_MODE=binary_mode)
 
-- url: FTP server, ex "localhost"
-- file_name: name of file to download
-- options: options for connection, ex use ssl, implicit security, etc.
-- save_path: location to save file to, if not specified file is written to a buffer
```

Comment:
Some of these arguments are still relevant and could be documented. See #4 in http://julia.readthedocs.io/en/latest/manual/documentation/#documentation